components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Settings.php

```diff
@@ -220,7 +246,7 @@ public function options( $options, $pod ) {
 				}
 
 				$options['pods-pfat']['pfat_archive']['data'] = array( null => __( 'No Archive view template', 'pods' ) ) + ( array_combine( $this->get_template_titles(), $this->get_template_titles() ) );
```

Comment:
Duplicate code (first part, I can't get multiline suggestion to work)
```suggestion
				$template_titles = array_combine( $this->get_template_titles(), $this->get_template_titles() );
				$options['pods-pfat']['pfat_archive']['data'] = array( null => __( 'No Archive view template', 'pods' ) ) + $template_titles;
```